Tigran Kirakosov/flexbox_version/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Dosis:200,300,400,600&display=swap"
+      rel="stylesheet"
+    />
+    <title>Tokyo Weather Widget</title>
+  </head>
+  <body>
+    <li class="container">

```

Comment:
Firstly - use `<li>` without `<ul>` , `<ol>` or `<menu>`  isn't correct - https://developer.mozilla.org/ru/docs/Web/HTML/Element/li

Reply:
Yes, that wasn't on purpose. Soon I will upload recent changes.